pages/public_cloud/containers_orchestration/managed-rancher-service/getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,276 @@
+8. Copy/Paste the provided credentials on the Rancher login page. Note: you will to change your password at first login. If you lose your password you have the possibility to generate a new one by using the "Generate access details" action button.
+
+You now have access to the Rancher dashboard, it is designed to offer an intuitive and comprehensive view of your containerized environment.
+This web-based interface serves as your command center for orchestrating containers & clusters, visualizing cluster health, and managing various aspects of your Kubernetes infrastructure. Explore the navigation menu, which includes sections for clusters, projects, applications, and Rancher settings.
+
+As it is a brand new Rancher instance you do not have any downstream Kubernetes clusters, next step is to add some of them. To do so you have two options: Import existing Kubernetes clusters or Create new clusters using Rancher.
+
+### Create or Import your Kubernetes clusters :
+#### Create a Kubernetes cluster with Rancher
+
+Using this option you will be able to create Kubernetes cluster from scratch. Rancher simplifies the creation of clusters by allowing you to create them through the Rancher UI rather than more complex alternatives.
+You can use Rancher to launch a Kubernetes cluster on any platform and location including:
+  - Hosted Kubernetes provider (ex: OVHcloud Managed Kubernetes Service, AWS EKS, GCP GKE,... )
+  - Infrastructure Provider - Public Cloud or Private Cloud (vSphere, Nutanix,...)
+  - Bare-metal servers, cloud hosted or on premise
+  - Virtual machines, cloud hosted or on premise
+
+
+For the last three options, when Rancher deploys Kubernetes onto these nodes, you can choose between Rancher Kubernetes Engine RKE2 or k3s distributions.
+Follow the official Rancher documentation on [How to launch Kubernetes with Rancher](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/launch-kubernetes-with-rancher) explaining how to define cluster settings, number of nodes (master, worker, etcd), authentication, and any additional configurations.
+
+we will detail bellow how to use OVHcloud as a Hosted Kubernetes provider and Infrastructure Provider using our official OVHcloud Drivers.
+
+
+##### Use OVHcloud as a **Hosted Kubernetes Provider**
+
+> [!warning]
+>
+> Deploying to OVHcloud will incur charges. For more information, refer to the [MKS](https://www.ovhcloud.com/en/public-cloud/prices/#568) and [Compute](https://www.ovhcloud.com/en/public-cloud/prices/) pricing pages.
+>
+
+  On this part we will detail how to use Rancher to create and manage [OVHcloud Managed Kubernetes Service](https://www.ovhcloud.com/en-gb/public-cloud/kubernetes/) clusters.
+
+  1. From the Rancher Homepage, click on **Create**
+  ![Rancher Homepage](images/rancher-homepage_create.png)
+
+  2. Use the OVHcloud **Hosted Kubernetes provider** by clicking on **OVHcloud MKS**
+  ![Cluster Creation](images/cluster-creation.png)
```

Comment:
```suggestion
  ![Cluster Creation](images/mks-cluster-creation.png)
```
And rename the screenshot ?

Reply:
I use this screenshot twice (mks and public cloud section), that's why